.golangci.yml

```diff
@@ -3,8 +3,6 @@
 linters-settings:
   gofmt:
     simplify: false
-  dupl:
```

Comment:
Can you insert this in a separate PR?

Reply:
Sure.

Reply:
After implementing your suggestion, this change is no longer needed.